### PR TITLE
Fix issue when username/password contain spaces

### PR DESCRIPTION
**Pull request by ifireball**
Fix issue #105

Signed-off-by: Barak Korren <bkorren@redhat.com>

### CODE REVIEW

**Comment by arilivigni** [comment]
[test]

**Comment by dirgim** [comment]
This looks good to me, but when I tried applying the same logic to [add_scc.yml#L6](https://github.com/CentOS-PaaS-SIG/contra-env-setup/blob/master/playbooks/roles/os_temps/tasks/add_scc.yml#L6), it failed.

File: playbooks/roles/os_temps/tasks/login_to_cluster.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
 ---
 # login to the cluster
 - name: "Login to the Openshift cluster"
-  shell: "{{ oc_bin }} login {{ cluster_ip }} --username={{ username }} --password={{ password }} --insecure-skip-tls-verify=true"
\ No newline at end of file
+  command: "{{ oc_bin }} login {{ cluster_ip|quote }} --username={{ username|quote }} --password={{ password|quote }} --insecure-skip-tls-verify=true"
```
[the remark this answers is not in the document]
Comment (ifireball) [reply playbooks/roles/os_temps/tasks/login_to_cluster.yml]: safer this way - less chance of injection, also I suspect `|quote` does not really work well with shell.

[the remark this answers is not in the document]
Comment (arilivigni) [reply playbooks/roles/os_temps/tasks/login_to_cluster.yml]: :+1:

**Comment by dirgim** [comment]
@ifireball It's fine, I got it working. I'm good to merge this.

**Pull request merged**